backend/server/tests/unit/test_schema.py

```diff
@@ -26,11 +28,15 @@ def setUp(self):
         self.matches = [
             FullMatchFactory(
                 year=year,
+                round_number=((idx % 23) + 1),
```

Comment:
is `idx`  -> index?

Reply:
Yeah, common naming convention in Python